src/variables.js

```diff
@@ -147,6 +153,9 @@ module.exports = {
 					this.store[key] = newValue
 					if (key === 'clips') {
 						this.updateVariable('clip_name', this.store.clips[this.store.clipID].name)
+						for (var i = 0; i < this.config.maxClips; i++) {
+							this.updateVariable('clip_' + (i+1), this.store.clips[i]?.name)
```

Comment:
I would prefer the variable names to be `clip_##_name`. There will be other clip related variables added soon (e.g. `clip_##_duration`).